src/ig/IGExporter.ts

```diff
@@ -267,7 +272,7 @@ export class IGExporter {
     // To work around this, we set the necessary permissions on executable
     // scripts after copying them to the IG path.
     try {
-      if (!this.isIgPubContext) {
+      if (!this.isIgPubContext || !this.isLegacyIgPubContext) {
```

Comment:
Shouldn't this be `!this.isIgPubContext && !this.isLegacyIgPubContext`? The way this is right now, I think as long as you are in one of the contexts, this if statement will be true, and we will try to `chmod` on files that don't actually exist, I get an error like this:
![image](https://user-images.githubusercontent.com/52747882/93921720-edc0db80-fcde-11ea-8bc5-d159121f05c5.png)

Reply:
Ugh yes. I did a bad job fixing this line a few hours ago. Should be fixed now.